products/firefox.md

```diff
@@ -14,11 +14,19 @@ auto:
 -   custom: true
 
 # For non-LTS versions, eol(x) = releaseDate(x+1)
+# Next planned ESR/LTS is 128 on 2024-07-09
+# Check ESR/LTS planing on https://wiki.mozilla.org/Release_Management/Calendar
```

Comment:
```suggestion
# For LTS version, eol(x) = releaseDate of the next major after the corresponding version last minor LTS on https://wiki.mozilla.org/Release_Management/Calendar, if available.
# Next planned ESR/LTS is 128 on 2024-07-09.
```

Don't know if it's clear, but it helps me understand how to ESR EOL dates are computed.

Reply:
The formulation is ok for me, will commit and merge.